blottertrax/helper/title_parser.py

```diff
@@ -103,7 +92,7 @@ def get_song_name_from_submission_title(cls, post_title, artist_name):
 
         # remove artist
         # "a-b (feat c) - -- - d [e/f](g)" -> " (feat c) - -- - d [e/f](g)"
-        post_title = post_title.rsplit(artist_name[0].lower(), 1)[1]
+        post_title = post_title.split(artist_name[0].lower(), 1)[1]
```

Comment:
rsplit was causing issues if the artist name appeared in the song name.

ex: "Tucky Buzzard - Tucky Buzzard (Full Album) [psych/rock] 1971"